solidity/ts/testsuite/simulator/utils/utilities.ts

```diff
@@ -177,12 +176,63 @@ export const mintCash = async (mockWindowEthereum: MockWindowEthereum, mintAmoun
 	await mockWindowEthereum.addStateOverrides({ [cashAddress]: { stateDiff: stateSets }})
 }
 
+export const mintRep = async (mockWindowEthereum: MockWindowEthereum, mintAmounts: { address: Address, amount: bigint }[]) => {
+	const repAddress = await getRepAddress(createWriteClient(mockWindowEthereum, VITALIK, 0))
+	const overrides = mintAmounts.map((mintAmount) => {
+		const encodedKeySlotHash = keccak256(encodeAbiParameters([{ type: 'address' }, { type: 'uint256' }], [mintAmount.address, 1n]))
+		return { key: encodedKeySlotHash, value: mintAmount.amount }
+	})
+	const stateSets = overrides.reduce((acc, current) => {
+		acc[current.key] = current.value
+		return acc
+	}, {} as { [key: string]: bigint } )
+	await mockWindowEthereum.addStateOverrides({ [repAddress]: { stateDiff: stateSets }})
+}
+
+export const getCashAddress = async (client: ReadClient) => {
+	return await client.readContract({
+		abi: ABIS.mainnet.universe,
+		functionName: 'cash',
+		address: addressString(AUGUR_UNIVERSE_ADDRESS),
+		args: []
+	})
+}
+
+export const getRepAddress = async (client: ReadClient) => {
+	return await client.readContract({
+		abi: ABIS.mainnet.universe,
+		functionName: 'getReputationToken',
+		address: addressString(AUGUR_UNIVERSE_ADDRESS),
```

Comment:
I think these functions should also take universe address as parameter as it might change